x-pack/plugins/cases/server/client/user_actions/stats.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { CaseUserActionStatsResponse } from '../../../common/api';
+import { CaseUserActionStatsResponseRt } from '../../../common/api';
+import { createCaseError } from '../../common/error';
+import type { CasesClientArgs } from '..';
+import type { UserActionGet } from './types';
+import type { CasesClient } from '../client';
+
+export const getStats = async (
+  { caseId }: UserActionGet,
+  casesClient: CasesClient,
+  clientArgs: CasesClientArgs
+): Promise<CaseUserActionStatsResponse> => {
+  const {
+    services: { userActionService },
+    logger,
+  } = clientArgs;
+
+  try {
+    await casesClient.cases.resolve({ id: caseId, includeComments: false });
```

Comment:
nit: I think we can do both of these calls in a `await Promise.all([])`, just a slight optimization